app/javascript/blacklight-frontend/checkbox_submit.js

```diff
@@ -44,7 +44,7 @@ export default class CheckboxSubmit {
         counter.innerHTML = json.bookmarks.count;
       });
 
-      var e = new CustomEvent('bookmark.blacklight', { detail: { checked: this.checked } });
+      var e = new CustomEvent('bookmark.blacklight', { detail: { checked: this.checked, document_id: this.formTarget.dataset.documentId } });
       window.dispatchEvent(e)
```

Comment:
```suggestion
      this.formTarget.dispatchEvent(e)
```
Instead of sending the document id, could we dispatch the event from the form and let consumers do whatever they need to do?